tests/test_diaForcedSource.py

```diff
@@ -37,7 +37,9 @@
 
 
 def create_test_dia_objects(n_points, wcs, startPos=100):
-    """Create dummy DIASources or DIAObjects for use in our tests.
+    """Create dummy DIASources or DIAObjects for use in our tests. Adds
+    Three test sources outsize of the ccd area.
```

Comment:
"outside" of the ccd area?

Reply:
Moved them to the setUp section and added a bit more clarity in the comments.